content/doc/developer/plugin-governance/adopt-a-plugin.adoc

```diff
@@ -18,25 +18,44 @@ This page addresses cases when there is no active maintainer of a plugin:
   In such case we can transfer ownership for plugins being hosted within the `jenkinsci` GitHub organization.
   Such transfer may take a while.
 
-== Requesting commit access
+== Plugins marked for adoption
 
-Email the http://jenkins-ci.org/content/mailing-lists[Jenkins Developers mailing list] and request to be made a maintainer 
-(which usually means commit access to the plugin repository and being made default assignee for newly reported issues in JIRA).
-Please provide the following things in the request:
+The procedure is two-fold: 
+
+**Announce your adoption intention by mail** 
+
+Send the notification email to the https://groups.google.com/g/jenkinsci-dev[Jenkins Developers mailing list].
+Be sure to provide the following information: 
 
 * Link to a plugin you want to adopt
+* The status of the plugin ("for adoption" or "abandoned")
 * Link(s) to pull requests you want to deliver, if applicable
 * Your GitHub username/id (e.g. oleg-nenashev for https://github.com/oleg-nenashev/)
 * Your Jenkins infrastructure account id. link:https://accounts.jenkins.io/[Create your account] if you don't have one.
+* The link to the "Repository Permission Updater" PR described below
+
+**Submit a PR on the "Repository Permission Updater" repository**
+
+File a pull request against link:https://github.com/jenkins-infra/repository-permissions-updater[Repository Permission updater] to be able to deploy snapshots and releases for your plugin.
```

Comment:
I don't have a strong opinion but I kind feel like there's not much point in mentioning snapshots, and I think it confuses people when they try and it doesn't show up in any release center?

Reply:
Good point. Will correct accordingly.